source/languages/en/riakcs/cookbooks/configuration/Configuring-Stanchion.md

```diff
@@ -8,60 +8,92 @@ audience: intermediate
 keywords: [operator, configuration]
 ---
 
-On the Stanchion node, you set the IP address and port for listening and
-the Riak IP address and port. On each Riak CS node, you have to set
-three Stanchion configuration properties.
+In your cluster, you must include one -- and only one -- Stanchion node. All the
+Riak CS nodes in that cluster must then be configured to communicate with that
+Stanchion node so that the cluster is able to track and negotiate
+causally-sensitive operations.
+
+All of the settings used by the Stanchion node are stored in the
+`stanchion.conf` file, which is located in the `/etc/riak-cs` folder on most
+operating systems.
+
+If you're upgrading from a version of Riak CS prior to 2.0.0 -- when the
+`stanchion.conf` and `riak-cs.conf` files was introduced -- you can still use
+the old-style `app.config` configuration files. Examples for both configuration
+types will be provided.
+
+```stanchionconf
+configuration.name = value
+```
+
+```appconfig
+{stanchion, [
+             %% Configs here
+            ]}
+```
 
 ## Specifying the Stanchion IP Address and Port
 
-If you have a single node, you don't have to change the settings for the
-address to listen on because Stanchion simply listens to the requests
-from the local host. If your Riak CS system has multiple nodes, you set
-the IP address and port that Stanchion listens on for requests from
-other nodes.
+If you have a single node, you don't have to change the Stanchion settings
+because Stanchion simply listens to the requests from the local host. If your
+Riak CS cluster has multiple nodes, you must set the IP address and port that
+Stanchion listens on for requests from other nodes.
 
-These settings reside in the Stanchion `app.config` file, which is
-located in the `/etc` folder on most operating systems. The settings
-appear in the `stanchion` config section of the file, which looks like
-this:
+You can set the IP using the `listener` parameter. Replace `127.0.0.1` with the
+IP address of the Stanchion node, and `8080` with the port of the Stanchion
+node.
+
+```stanchionconf
+listener = 127.0.0.1:8080
+```
 
 ```appconfig
 {stanchion, [
-
-    %% Configs here
-
-]}
+             {host, {"127.0.0.1", 8080}},
```

Comment:
Correct. Maybe 8080 is also used for CS default listening port, so 8085 might be more suitable?